src/CMakeLists.txt

```diff
@@ -175,6 +198,7 @@ foreach(ccfile ${CYCLUS_COIN_SRC})
 endforeach()
 list(REMOVE_ITEM cc_files ${CMAKE_CURRENT_SOURCE_DIR}/coin_helpers.cc)
 list(REMOVE_ITEM cc_files ${CMAKE_CURRENT_SOURCE_DIR}/prog_translator.cc)
+list(REMOVE_ITEM cc_files ${CythonModuleSrc})
```

Comment:
Can you add a comment to indicate why we need to remove these, for posterity?

Reply:
Comment added